server/src-lib/Hasura/RQL/Types/RemoteSchema.hs

```diff
@@ -94,3 +113,11 @@ validateRemoteSchemaDef (RemoteSchemaDef mUrl mUrlEnv hdrC fwdHdrs mTimeout) =
     hdrs = fromMaybe [] hdrC
 
     timeout = fromMaybe 60 mTimeout
+
+data DropRemoteSchemaPermissions
+  = DropRemoteSchemaPermissions
+  { drsPermRemoteSchema :: RemoteSchemaName
+  , drsPermRole         :: RoleName
```

Comment:
Use `permission` instead of `perm` as it reflects in JSON interface.

Reply:
`perm` is dropped in the json